mgenepy/utils/helper.py

```diff
@@ -87,6 +86,10 @@ def generateGeneNames(
 
 
 def _fetchFromServer(ensemble_server, attributes):
```

Comment:
I'm confused by the use of Biomart. I thought we switched to using an archived table for this so that it'll be stable over time. I moved the import down so that this module imports fine without the biomart library, but if we call `_fetchFromServer`, it'll fail at that time.